### PR TITLE
[ACS-5183] properties facet file size and file type

### PR DESCRIPTION
**Pull request by AleksanderSklorz**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ACS-5183

**What is the new behaviour?**
Added properties facet which allows search by file size and type.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
ADF PR: https://github.com/Alfresco/alfresco-ng2-components/pull/8766

### CODE REVIEW

File: e2e/protractor/protractor.excludes.json [hunk 1]
```diff
@@ -6,13 +6,7 @@
   "C286252": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C284666": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C286269": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
-  "C279191": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279192": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279193": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279195": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
   "C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
```
Comment (AleksanderSklorz) [inline e2e/protractor/protractor.excludes.json:9]: @MichalKinas some of e2es can't be fixed without changes for your jira. Can we fix them later?

Comment (MichalKinas) [reply e2e/protractor/protractor.excludes.json:9]: Yes no worries

Comment (MichalKinas) [reply e2e/protractor/protractor.excludes.json:9]: Let's just add some comment or task for it not to forget about it

Comment (AleksanderSklorz) [reply e2e/protractor/protractor.excludes.json]: @MichalKinas thanks, I just raised jira for that and corrected link in excluded list to indicate on that jira 

https://alfresco.atlassian.net/browse/ACS-5632

File: projects/aca-content/assets/i18n/ar.json [hunk 2]
```diff
@@ -486,7 +486,7 @@
   },
   "INFO_PANEL": {
     "TABS": {
-    
+
```
Comment (DenysVuika) [inline projects/aca-content/assets/i18n/ar.json:489]: empty line probably not needed

File: projects/aca-content/assets/i18n/cs.json [hunk 3]
```diff
@@ -486,7 +486,7 @@
   },
   "INFO_PANEL": {
```
Comment (DenysVuika) [inline projects/aca-content/assets/i18n/cs.json:487]: we probably need to remove the whole section as it's empty

File: projects/aca-testing-shared/src/components/search/filters/properties-filter.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { GenericFilter } from './generic-filter';
```
Comment (DenysVuika) [inline projects/aca-testing-shared/src/components/search/filters/properties-filter.ts:1]: license header missing

File: projects/aca-content/assets/i18n/cs.json (hunk 3 above)
Comment (AleksanderSklorz) [reply projects/aca-content/assets/i18n/cs.json:487]: removed whole empty TABS block

File: projects/aca-testing-shared/src/components/search/filters/properties-filter.ts (hunk 4 above)
Comment (AleksanderSklorz) [reply projects/aca-testing-shared/src/components/search/filters/properties-filter.ts:25]: done

File: projects/aca-content/assets/i18n/ar.json (hunk 2 above)
Comment (AleksanderSklorz) [reply projects/aca-content/assets/i18n/ar.json]: removed whole TABS block so no empty line anymore

**Review by MichalKinas — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

**Pull request merged**